modules/solid_properties/doc/content/modules/solid_properties/index.md

```diff
@@ -100,18 +100,53 @@ The userobjects defining the material properties are set up in the `UserObjects`
 example, to use stainless steel 316 thermal properties to provide the thermal conductivity in
 the `HeatDiffusion` kernel above, the input file syntax would be:
 
-!listing modules/solid_properties/test/tests/userobjects/stainless_steel_316/stainless_steel_316.i
-  start=Modules
-  end=Kernels
+```
+[Modules]
+  [SolidProperties]
+    [steel]
+      type = ThermalSS316Properties
+    []
+  []
+[]
+
+[Materials]
+  [sp_mat]
+    type = ThermalSolidPropertiesMaterial
+    temperature = T
+    sp = steel
+    output_properties = 'thermal_conductivity density specific_heat'
```

Comment:
I'm okay with the merging of the test files - but for these examples, I think we should make them as simple as possible to eliminate any potential issues with future syntax changes. i.e. I think the `output_properties` and `outputs` lines should be removed in these snippets since they were only around for exodiff purposes.

Reply:
🤦 Yep, that was a copy-paste error. I'll fix it!